leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -49,6 +51,7 @@
     <activity
         android:theme="@style/leak_canary_Theme.Transparent"
         android:name=".internal.RequestStoragePermissionActivity"
+        android:process=":leakcanary"
```

Comment:
Being mindful of the current OOM issues we've been seeing, it might be better to move these to yet another process.  WDYT?